loader/gpa_helper.c

```diff
@@ -273,7 +273,6 @@ void *globalGetProcAddr(const char *name) {
     if (!strcmp(name, "EnumerateInstanceExtensionProperties")) return vkEnumerateInstanceExtensionProperties;
     if (!strcmp(name, "EnumerateInstanceLayerProperties")) return vkEnumerateInstanceLayerProperties;
     if (!strcmp(name, "EnumerateInstanceVersion")) return vkEnumerateInstanceVersion;
-    if (!strcmp(name, "GetInstanceProcAddr")) return vkGetInstanceProcAddr;
```

Comment:
Does this not break the `vkGetInstanceProcAddr(NULL, "vkGetInstanceProcAddr")` case?

The logic is that if a function isn't a global function (which with this change vkGetInstanceProcAddr is not) then it MUST have a non-NULL instance handle.